### PR TITLE
Fixed test logic for patching domain

### PR DESCRIPTION
**Pull request by marinakog**
fixed test, so patching domain should not fail, the test fails because of OWLS-102288

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/12722

### CODE REVIEW

**Review by anpanigr — COMMENTED** [review]
The test ItKubernetesDomainEvents/testDomainK8sEventsScalePastMaxAndChangeIntrospectVersion still fails

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesDomainEvents.java [hunk 1]
```diff
@@ -422,7 +422,7 @@ void testDomainK8sEventsScalePastMaxAndChangeIntrospectVersion() {
           + "]";
 
       logger.info("Updating introspect version  using patch string: {0}",  patchStr);
-      assertFalse(patchDomainCustomResource(domainUid, domainNamespace3, new V1Patch(patchStr),
+      assertTrue(patchDomainCustomResource(domainUid, domainNamespace3, new V1Patch(patchStr),
```
[the remark this answers is not in the document]
Comment (marinakog) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesDomainEvents.java:425]: fixed

**Comment by marinakog** [comment]
> The test ItKubernetesDomainEvents/testDomainK8sEventsScalePastMaxAndChangeIntrospectVersion still fails

yes, see OWLS-102288, I filed new jira

**Review by anpanigr — APPROVED** [review]
lgtm

**Pull request merged**